lib/releaselib.sh

```diff
@@ -1099,6 +1101,91 @@ release::docker::release () {
   return 0
 }
 
+# TODO(vdf): Consider collapsing this into release::docker::release and renaming
+#            that function AFTER the k8s.gcr.io Vanity Domain Flip (VDF).
+###############################################################################
+# Validates that image manifests have been pushed to a specified remote registry.
+# Uses 'skopeo inspect'.
+#
+# @param registry - docker registry
+# @param version - version tag
+# @param build_output - build output directory
+# @return 1 on failure
+release::docker::validate_remote_manifests () {
```

Comment:
I know in other places in this file it's not being checked but when you are writing new one you should check if the function receives enough arguments it expects and if not fail.

Reply:
Thanks! I've added `common::argc_validate` to each function. :)